arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/call/precompileSubsection/PrecompileSubsection.java

```diff
@@ -209,7 +209,9 @@ public long returnDataSize() {
   }
 
   public Bytes rawCallerMemory() {
-    return callSection.getCallDataRange().getRawData();
```

Comment:
This is the fix after @OlivierBBB 's comment: to get the rawCallerMemory, if the callDataRange is empty, then we check in the returnAtRange